src/utils.tsx

```diff
@@ -1 +1,12 @@
+export type LocalizedStringMap = {
+  [lang: string]: string;
+};
+
 export const gettext = (text: string) => text;
+
+export const getLocalizedString = (
+  localizedStringMap: LocalizedStringMap,
+  lang = process.env.REACT_APP_DEFAULT_API_LANG as string,
```

Comment:
Does TS understand from this that the type of `lang` is `string` but it is also optional (e.g., like `lang?: string`)?

Reply:
It understands that there is a default value, making the argument optional. Because it cannot infer what's in `process.env`, we have to hint it by adding `as string` (it considers everything into `process.env` to be optional strings). If we were to write `'en-US'`, it would have inferred the correct type.

Reply:
So then TS doesn't necessarily know that we want to restrict `lang` to only accept strings? I'm just trying to understand if this argument is typed or not.